src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-Guardrails-KeyVault.json

```diff
@@ -113,7 +113,7 @@
                     "displayName": "The minimum days before expiry",
                     "description": "Enter the days before expiration of the certificate when you want to trigger the policy action. For example, to trigger a policy action 90 days before the certificate's expiration, enter '90'."
                 },
-                "defaultValue": 90
+                "defaultValue": 20
```

Comment:
was this intended to change?

Reply:
I aligned with what FSI has set as a standard. We were never really specific/deliberate in the defaults we set, and 90 days seems overly aggressive (noise easily ignored - 20 days is a concern and requires immediate action).